dkg/pedersen/reencrypt.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 type onChainSecret struct {
-	U    kyber.Point // U is the random part of the encrypted secret
+	K    kyber.Point // K is the random part of the encrypted secret
```

Comment:
K is typically used for keys. Why not keep `U` (or something else) for randomness ?